cpp/ps0004/ps0004_soin_cpp.cpp

```diff
@@ -2,17 +2,27 @@
 #include <conio.h>
 using namespace std;
 int main() {
-	int num;
+	// prompting user for number 1/1.5
+	float num;
 	cout << "Enter a number (ASCII value between 32 and 126) \n";
 	cin >> num;

```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input